src/GeneralStateTestsFiller/stCreateTest/CREATE_HighNonceDelegatecallMinus1Filler.yml

```diff
@@ -0,0 +1,156 @@
+# The test calls CREATE from an account with max allowed nonce - 1.
+CREATE_HighNonceDelegatecallMinus1:
+  env:
+    currentCoinbase: 2adc25665018aa1fe0e6bc666dac8fc2697ff9ba
+    currentDifficulty: '0x020000'
+    currentGasLimit: '89128960'
+    currentBaseFee: '10'
+    currentNumber: '1'
+    currentTimestamp: '1000'
+    previousHash: 5e20a0453cecd065ea59c37ac63e079ee08998b6045136a8ce6635c7912ec0b6
+
+  pre:
+    a94f5374fce5edbc8e2a8697c15331677e6ebf0b:
+      balance: 1000000000
+      code: ''
+      nonce: 0
+      storage: {}
+
+    000000000000000000000000fffffffffffffffe:
+      balance: 0
+      code: |
+        :yul
+        {
+          // initcode: { return(0, 1) }
+          mstore(0, 0x60016000f3000000000000000000000000000000000000000000000000000000)
+          mstore(0, create(0, 0, 5))
+          return(0, 32)
+        }
+      nonce: '0xfffffffffffffffe'
+      storage: {}
+
+    000000000000000000000000ffffffffffffffff:
+      balance: 0
+      code: |
+        :yul
+        {
+          // initcode: { return(0, 1) }
+          mstore(0, 0x60016000f3000000000000000000000000000000000000000000000000000000)
+          mstore(0, create(0, 0, 5))
+          return(0, 32)
+        }
+      nonce: '0xffffffffffffffff'
+      storage: {}
+
+    b94f5374fce5edbc8e2a8697c15331677e6ebf0b:
+      balance: 0
+      code: |
+        :yul
+        {
+          let calltype      := calldataload(4)
+          let callcontract  := calldataload(36)
+          let callcount     := 2
+
+          if eq(calltype, 0) {
+            for { let i := 0 } lt(i, callcount) { i := add(i, 1) } {
+              pop(delegatecall(sub(gas(), 1000), callcontract, 0, 0, 0, 32))
+              let result := mload(0)
+              sstore(i, result)
+            }
+          }
+          if eq(calltype, 1) {
+            for { let i := 0 } lt(i, callcount) { i := add(i, 1) } {
+              pop(callcode(sub(gas(), 1000), callcontract, 0, 0, 0, 0, 32))
+              let result := mload(0)
+              sstore(i, result)
+            }
+          }
+          if eq(calltype, 2) {
+            for { let i := 0 } lt(i, callcount) { i := add(i, 1) } {
+              pop(call(sub(gas(), 1000), callcontract, 0, 0, 0, 0, 32))
+              let result := mload(0)
+              sstore(i, result)
+            }
+          }
+        }
+      nonce: '0xfffffffffffffffe'
+      storage: {}
+
+  transaction:
+    data:
+    # Parameters:
+    #  1) Call type:
+    #       0 - delegateCall
+    #       1 - callCode
+    #       2 - call
+    #  2) Called contract nonce
+    # - Contract A (Nonce=0xfffffffffffffffe) delegate calls Contract B (Nonce=0xfffffffffffffffe), contract create first suceeds, then fails
+      - ':label a_max_nonce_minus_1_delegate_call_create :abi f(uint,uint) 0x00 0xfffffffffffffffe'
```

Comment:
you can make it even more straight foward. 

```
1)  call type
2)  caller nonce  (A)
3)  destination nonce (B)
4) create/create2 for the destiantion
```

reveiver contract parse the args.  (transaction destiantion)
and make a call to callernonce (A) contact. passing the calltype and destination and create type as argument
then callernonce contract (A) do the calltype to destination (B)  and destination (B) do the create/create2
after each operation to mark success callernonce(A)  stores 1:1,  destination B stores 2:2 to check in expect section
also check nonce in expect section for both all variants of A, and B, and storage at cells 1,2 for all types of A and B

Reply:
I did all these changes in latest push with some minor differences:
- callernonce (A) is always the same contract, but for the test cases where callernonce == max nonce, the contract first does a dummy CREATE to increase its nonce before doing any delegate call/call code/call.
- The store actions store the resulting contract address instead of a constant 1 or 2.
Let me know if this is ok.